tapdb/universe_stats.go

```diff
@@ -575,14 +575,48 @@ func (u *UniverseStats) AggregateSyncStats(
 
 	log.Debugf("Populating aggregate sync stats")
 
-	dbStats, err := u.querySyncStats(ctx)
-	if err != nil {
+	var (
+		resChan = make(chan universe.AggregateStats, 1)
+		errChan = make(chan error, 1)
+		dbStats universe.AggregateStats
```

Comment:
nit: move this just before the `select`, otherwise it might look like it's shadowed in the Goroutine.